bin/katana/src/cli/node.rs

```diff
@@ -48,174 +47,228 @@ use tracing_log::LogTracer;
 use tracing_subscriber::{fmt, EnvFilter};
 use url::Url;
 
-use crate::utils::{parse_block_hash_or_number, parse_genesis, parse_seed};
+use crate::utils::{parse_block_hash_or_number, parse_genesis, parse_seed, LogFormat};
 
 #[derive(Parser, Debug)]
 pub struct NodeArgs {
+    /// Don't print anything on startup.
     #[arg(long)]
-    #[arg(help = "Don't print anything on startup.")]
     pub silent: bool,
 
+    /// Disable auto and interval mining, and mine on demand instead via an endpoint.
     #[arg(long)]
     #[arg(conflicts_with = "block_time")]
-    #[arg(help = "Disable auto and interval mining, and mine on demand instead via an endpoint.")]
     pub no_mining: bool,
 
+    /// Block time in milliseconds for interval mining.
     #[arg(short, long)]
     #[arg(value_name = "MILLISECONDS")]
-    #[arg(help = "Block time in milliseconds for interval mining.")]
     pub block_time: Option<u64>,
 
+    /// Directory path of the database to initialize from.
+    ///
+    /// The path must either be an empty directory or a directory which already contains a
+    /// previously initialized Katana database.
     #[arg(long)]
     #[arg(value_name = "PATH")]
-    #[arg(help = "Directory path of the database to initialize from.")]
-    #[arg(long_help = "Directory path of the database to initialize from. The path must either \
-                       be an empty directory or a directory which already contains a previously \
-                       initialized Katana database.")]
     pub db_dir: Option<PathBuf>,
 
-    #[arg(long = "fork.rpc-url", value_name = "URL", alias = "rpc-url")]
-    #[arg(help = "The Starknet RPC provider to fork the network from.")]
-    pub fork_rpc_url: Option<Url>,
-
-    #[arg(long = "fork.block", value_name = "BLOCK_ID", alias = "fork-block-number")]
-    #[arg(requires = "fork_rpc_url")]
-    #[arg(help = "Fork the network at a specific block id, can either be a hash (0x-prefixed) \
-                  or number.")]
-    #[arg(value_parser = parse_block_hash_or_number)]
-    pub fork_block: Option<BlockHashOrNumber>,
-
-    #[arg(long)]
-    pub dev: bool,
-
-    #[arg(long)]
-    #[arg(help = "Output logs in JSON format.")]
-    pub json_log: bool,
-
-    /// Enable Prometheus metrics.
+    /// Configure the messaging with an other chain.
     ///
-    /// The metrics will be served at the given interface and port.
-    #[arg(long, value_name = "SOCKET", value_parser = parse_socket_address, help_heading = "Metrics")]
-    pub metrics: Option<SocketAddr>,
-
+    /// Configure the messaging to allow Katana listening/sending messages on a
+    /// settlement chain that can be Ethereum or an other Starknet sequencer.
+    /// The configuration file details and examples can be found here: <https://book.dojoengine.org/toolchain/katana/reference#messaging>
```

Comment:
This link is no longer active. better removing from the code for now until we have a more stable documentation links. 👍

Reply:
ah didnt check it first

Reply:
removed in [26c6a2b](https://github.com/dojoengine/dojo/pull/2663/commits/26c6a2b99008aa1f3e96fe7c57940cacbe53a68f)